### PR TITLE
docs: update play permissions

### PR DESCRIPTION
**Pull request by yT0n1**
closes https://github.com/camunda/play/issues/593

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/modeler/web-modeler/play-your-process.md [hunk 1]
```diff
@@ -155,7 +155,8 @@ For more information about terms, refer to our [licensing and terms page](https:
 
 ### Camunda 8 SaaS
 
-In Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
+In Camunda 8 SaaS, Play is available to all Web Modeler users with commenter, editor, or admin permissions within a project.
+Additionally, within their organization, users need to have a [role](/components/console/manage-organization/manage-users.md#roles-and-permissions) which has deployment privileges.
 Enterprise users need an admin to enable Play by opting in to [alpha features](/components/console/manage-organization/enable-alpha-features.md).
```
Comment (huygur) [inline docs/components/modeler/web-modeler/play-your-process.md:160]: shouldn't we remove this as well? 🤔

Comment (yT0n1) [reply docs/components/modeler/web-modeler/play-your-process.md:160]: That would have been my follow-up question, too. There is also the reference to alpha in Self-Managed. I'd rather create a ticket to clean this up on one swoop, but I can also remove the line here. wdyt?

Comment (huygur) [reply docs/components/modeler/web-modeler/play-your-process.md:160]: whatever you prefer :)

Comment (yT0n1) [reply docs/components/modeler/web-modeler/play-your-process.md:160]: Seperat ticket ;)

**Review by huygur — APPROVED** [review]

**Pull request merged**